design/workitems.go

```diff
@@ -101,9 +101,8 @@ var workItemReorder = Reorder(
 	workItem,
 	position)
 
-// new version of "list" for migration
+// endpoints that DO NOT depend on the space id (ie, when the work item ID is specified in the URL, there's no need to pass the space ID)
 var _ = a.Resource("workitem", func() {
-	a.Parent("space")
 	a.BasePath("/workitems")
```

Comment:
I think we should keep the plural form in all cases, that's a convention in API naming.